Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Utils/MapMembersUtils.cs

```diff
@@ -0,0 +1,44 @@
+using ArcGIS.Desktop.Internal.Mapping;
+using ArcGIS.Desktop.Mapping;
+using ArcLayer = ArcGIS.Desktop.Mapping.Layer;
+
+namespace Speckle.Connectors.ArcGIS.Utils;
+
+public class MapMembersUtils
+{
+  public List<MapMember> GetAllMapMembers(Map map)
+  {
+    // first get all map layers
+    Dictionary<MapMember, int> membersIndices = new();
+    int count = 0;
+    var layers = map.Layers;
+    count = UnpackMapLayers(membersIndices, layers, count);
+
+    return membersIndices.Select(x => x.Key).ToList();
```

Comment:
I'm confused. We are not returning count info, and we are not using the count info as condition then why we track count at all?